models/models.js

```diff
@@ -18,10 +18,23 @@ exports.fetchReviewById = (review_id) => {
         if (!review.rows[0]) {
             return Promise.reject({
                 status: 404,
-                msg: `Review ${review_id} could not be found`,
+                msg: 'Not Found',
             }
             );
         }
         return review.rows;
     });
+};
+
+exports.fetchCommentsByReviewId = (review_id) => {
+    return db.query('SELECT * FROM comments WHERE review_id = $1 ORDER BY created_at ASC;', [review_id]).then(comments => {
+        if (!comments.rows[0]) {
```

Comment:
We only want to fail here if it's because the id is valid but non-existent. If the id is valid but has no associated comments, we wouldn't send back an error